Upstream: no (see Reason)
Reason: We are installing some llvm tools to a nonstandard path, which we add
        to $PATH in src_test. This problem doesn't surface with building clang
        alongside llvm as upstream suggests.

From d0bc58669cc926d0e9b9662dd6e3e58d04153da3 Mon Sep 17 00:00:00 2001
From: Johannes Nixdorf <mixi@exherbo.org>
Date: Tue, 13 Jan 2015 21:09:39 +0100
Subject: [PATCH] test/lit.cfg: use $PATH to find the llvm tools

---
 test/lit.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/lit.cfg b/test/lit.cfg
index 846a6e9..b9370bb 100644
--- a/test/lit.cfg
+++ b/test/lit.cfg
@@ -300,7 +300,7 @@ config.substitutions.append(
 # with the full path to the build directory holding that tool.  This
 # ensures that we are testing the tools just built and not some random
 # tools that might happen to be in the user's PATH.
-tool_dirs = os.path.pathsep.join((clang_tools_dir, llvm_tools_dir))
+tool_dirs = os.path.pathsep.join((clang_tools_dir, llvm_tools_dir, config.environment['PATH']))
 
 # Regex assertions to reject neighbor hyphens/dots (seen in some tests).
 # For example, don't match 'clang-check-' or '.clang-format'.
-- 
2.2.1

